src/analyze.js

```diff
@@ -318,7 +318,7 @@ module.exports = async function (id, code, job) {
     if (!wildcardPattern.endsWith('*'))
       wildcardPattern += '?(' + (job.ts ? '.ts|' : '') + '.js|.json|.node)';
 
-    if (job.ignoreFn(job.base ? path.relative(job.base, wildcardDirPath + wildcardPattern) : wildcardDirPath + wildcardPattern, id))
+    if (job.ignoreFn(path.relative(job.base, wildcardDirPath + wildcardPattern)))
```

Comment:
```suggestion
    if (job.ignoreFn(path.relative(job.base, join(wildcardDirPath, wildcardPattern))))
```

Should this be a path.join() instead of string concat?

Reply:
It doesn't matter since `wildcardDirPath` has already been through `path.resolve`.

Reply:
What about Windows support?

Reply:
`path.relative` would effectively normalize that out for us.